app/_hub/kong-inc/app-dynamics/overview/_index.md

```diff
@@ -85,6 +85,9 @@ for more information about the configuration parameters.
 | `KONG_CONTROLLER_CERTIFICATE_FILE` | Path to a self-signed certificate file. For example, `/etc/kong/certs/ca-certs.pem`. | String | | 
 | `KONG_CONTROLLER_CERTIFICATE_DIR` | Path to a certificate directory. For example, `/etc/kong/certs/`. | String | | 
 {% endif_version %}
+{% if_version gte:3.8.x %}
+| `KONG_ANALYTICS_ENABLE` | Whether to enable sending analytics to AppDynamics. | Boolean | `false` | 
```

Comment:
Please follow the appdynamics library doc describing it. https://docs.appdynamics.com/appd/23.x/latest/en/application-monitoring/install-app-server-agents/c-c++-sdk/c-c++-sdk-reference#id-.CCPPSDKReferencev23.1-appd_config_set_analytics_enabled